refinery/core/models.py

```diff
@@ -364,10 +367,36 @@ class SharableResource(OwnableResource):
     def __unicode__(self):
         return self.name
 
+    def get_owner(self):
```

Comment:
Moved from the data_set class. Fritz noted this method improved performance 10%-20% percent.